android/app/src/main/java/edu/gatech/cs/environmentalodors/MapsActivity.java

```diff
@@ -247,6 +248,7 @@ public void generateFakeData() {
         LatLng center = DEFAULT_LOCATION; // approximately atlanta
         float radius = 0.2f;
         int reportCount = 5;
+        int commentCount = 5;
```

Comment:
I think it'll be fine this way - it allows easy editing of the test without having to sift through the code.  That is probably a good policy for non-tests though.